contracts/seed/SeedFactory.sol

```diff
@@ -44,35 +43,38 @@ contract SeedFactory is CloneFactory {
     }
 
     modifier protected() {
-        require(initialized,                    "SeedFactory: contract not initialized");
-        require(msg.sender == address(avatar),  "SeedFactory: protected operation");
+        require(initialized, "SeedFactory: contract not initialized");
+        require(
+            msg.sender == address(avatar),
+            "SeedFactory: protected operation"
+        );
         _;
     }
 
     /**
-      * @dev           Initialize proxy.
-      * @param _avatar The address of the Avatar controlling this contract.
-      * @param _parent The address of the Seed contract which will be a parent for all of the clones.
-      */
+     * @dev           Initialize proxy.
+     * @param _avatar The address of the Avatar controlling this contract.
+     * @param _parent The address of the Seed contract which will be a parent for all of the clones.
+     */
     function initialize(Avatar _avatar, Seed _parent) external initializer {
         require(_avatar != Avatar(0), "SeedFactory: avatar cannot be null");
-        require(_parent != Seed(0),   "SeedFactory: parent cannot be null");
+        require(_parent != Seed(0), "SeedFactory: parent cannot be null");
```

Comment:
let's be consistent with the https://github.com/PrimeDAO/contracts/blob/main/contracts/incentives/FarmFactory.sol naming and rename `parent` to `masterCopy` through the code

Reply:
Okay, will do that.